{"uuid": "8a83593d-cea2-416a-af78-683c6034fd05", "children": ["6ef20714-418a-4587-bc6f-8d286525e542"], "befores": [{"name": "driver", "status": "passed", "start": 1691765102751, "stop": 1691765104437}], "afters": [{"name": "driver::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 917, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\conftest.py\", line 15, in driver\n    if request.node.rep_call.failed:\n       ^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1691765111346, "stop": 1691765111346}], "start": 1691765102751, "stop": 1691765111346}